HdProduction.HelpDesk.Api/Controllers/UsersController.cs

```diff
@@ -10,16 +12,31 @@ namespace HdProduction.HelpDesk.Api.Controllers
     public class UsersController
     {
         private readonly IUserService _userService;
+        private readonly IMapper _mapper;
 
-        public UsersController(IUserService userService)
+        public UsersController(IUserService userService, IMapper mapper)
         {
             _userService = userService;
+            _mapper = mapper;
         }
 
         [HttpPost("")]
-        public Task Create(UserRequestModel requestModel)
+        public async Task<UserResponseModel> Create(UserRequestModel requestModel)
         {
-            return _userService.CreateAsync(requestModel.Email, requestModel.PwdHash);
+            var user = await _userService.CreateAsync(
+                requestModel.FirstName,
+                requestModel.LastName,
+                requestModel.Email,
+                requestModel.PwdHash
+            );
+            return _mapper.Map<User, UserResponseModel>(user);
```

Comment:
we usually do not return user object from service on create, update etc.
prefer return id from Create method and then call Find(id)
ef caches entity inside scope, so there won't be any overhead